api/models.go

```diff
@@ -85,7 +85,7 @@ type RPCTransaction struct {
 	Hash                common.Hash       `json:"hash"`
 	Input               hexutil.Bytes     `json:"input"`
 	Nonce               hexutil.Uint64    `json:"nonce"`
-	To                  *common.Address   `json:"to"`
+	To                  string            `json:"to"`
```

Comment:
I think this will create issues with the clients, because every type that is a pointer, should serialize to `null`, in JSON.
```shell
to: DATA, 20 Bytes - address of the receiver. null when its a contract creation transaction.
```